packages/grafana-ui/src/components/ClipboardButton/ClipboardButton.test.tsx

```diff
@@ -23,8 +27,8 @@ describe('ClipboardButton', () => {
     Object.assign(window, originalWindow);
   });
 
-  // TODO: flaky https://github.com/grafana/grafana/issues/105102
-  it.skip('should copy text to clipboard when clicked', async () => {
+  it('should copy text to clipboard when clicked', async () => {
+    jest.useFakeTimers();
```

Comment:
minor nit: the `useFakeTimers()`/`useRealTimers()` calls should probably either go in the `beforeAll`/`afterAll` blocks, or create a new isolated describe with its own `beforeAll`/`afterAll` that includes them for this test

reasoning: if an expectation in this test fails, it won't reach the `useRealTimers()` call, then fake timers will still be set for all other tests in this file (if there were any...). if it's in a `beforeAll`/`afterAll` somewhere we guarantee they're both called